.github/workflows/check-linked-issues.yml

```diff
@@ -1,6 +1,6 @@
 name: Check linked issues
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
we will have to do this change in ALL of our repositories. you can easily search for it because the workflow is called check-linked-issues.yml everywhere